clang/docs/ReleaseNotes.rst

```diff
@@ -259,6 +259,10 @@ Bug Fixes in This Version
   operator.
   Fixes (#GH83267).
 
+- Clang no longer fails to codegen static ``const`` variables whose initialiser performs
+  a floating-point division by 0 in C23.
+  Fixes (#GH84784).
```

Comment:
We usually only add release notes for things that are fixed between releases.  This wasn't broken in 18, so I don't think we need a release note.

Reply:
Oh, I thought it was, my bad. Yeah, we don’t need a release note then ofc.